source/fundamentals/enterprise-auth.txt

```diff
@@ -10,7 +10,7 @@ Enterprise Authentication Mechanisms
    :depth: 2
    :class: singlecol
 
-In this guide, we show you how to authenticate with MongoDB using each
+In this guide, you can learn how to authenticate in the MongoDB Java driver using each
```

Comment:
```suggestion
In this guide, you can learn how to authenticate with MongoDB using each
```
Issue: I think this was correct before. I think you authenticate with the server (need to log into server), rather than with the driver.